pkg/storage/system_intake.go

```diff
@@ -618,3 +618,16 @@ func (s *Store) GetMySystemIntakes(ctx context.Context) ([]*models.SystemIntake,
 
 	return intakes, err
 }
+
+// GetSystemIntakesWithGRBReviewHalfwayThrough retrieves all system intakes where the GRB review is halfway through
+func GetSystemIntakesWithGRBReviewHalfwayThrough(ctx context.Context, np sqlutils.NamedPreparer, logger *zap.Logger) ([]*models.SystemIntake, error) {
+	var intakes []*models.SystemIntake
+
+	err := namedSelect(ctx, np, &intakes, sqlqueries.SystemIntake.GetWhereGRBReviewIsHalfwayThrough, map[string]interface{}{})
```

Comment:
more of a curiosity - does this work? or does it cause errors? if it errors, then i will fix those functions. i should have allowed for `nil` there

```suggestion
	err := namedSelect(ctx, np, &intakes, sqlqueries.SystemIntake.GetWhereGRBReviewIsHalfwayThrough, nil)
```

Reply:
Yeah! Nil  doesn't function for this. I started with nil, but switched to the empty map for functionality